Ginger/GingerCoreCommon/Repository/SolutionRepository.cs

```diff
@@ -46,22 +46,22 @@ public class SolutionRepository
         /// </summary>
         private static List<string> mSolutionPathsToAvoid = new List<string>()
         {
-             "AutoSave",
-             "Recover",
-             "RecentlyUsed.dat",
-             "Backups",
-             "ExecutionResults",
-             "HTMLReports",
-
-             @"SharedRepository\Activities\PrevVersions",
-             @"SharedRepository\Actions\PrevVersions",
-             @"SharedRepository\Variables\PrevVersions",
-             @"SharedRepository\ActivitiesGroup\PrevVersions",
-
-             @"SharedRepository\Activities\PrevVerions",
-             @"SharedRepository\Actions\PrevVerions",
-             @"SharedRepository\Variables\PrevVerions",
-             @"SharedRepository\ActivitiesGroup\PrevVerions"
+             @"AutoSave\",
```

Comment:
Please verify this changes is not affecting SVN source control, there also it should avoid this folders and files

Reply:
Added check for Directories in SVN and appended backslash when necessary